corehq/apps/cloudcare/touchforms_api.py

```diff
@@ -61,7 +61,7 @@ def get_full_context(self, root_extras=None, session_extras=None):
         # always tell touchforms to include footprinted cases in its case db
         session_data["additional_filters"] = {"footprint": True}
         session_data.update(session_extras)
-        xform_url = reverse("xform_player_proxy")
+        xform_url = root_extras.get('formplayer_url')
```

Comment:
i checked and doesn't look like the method where this change happened ever gets called in the sms or ivr form workflows. but i ran the test suites for both to be sure and they passed, so we're good